wearipedia/devices/polar/vantage_synthetic.py

```diff
@@ -165,17 +178,15 @@ def continuity(x):
             "hrAvg": random_hr_avg(""),
             "calories": int(random_calories(duration)),
             "note": " ",
-            "sportName": np.random.choice(
-                ["Running", "Cycling", "Strength_Training", "Swimming"]
-            ),
+            "sportName": selected_activity,
             "sportId": int(np.random.choice([1, 2, 15, 16])),
             "startDate": d.strftime("%Y-%m-%d")
```

Comment:
For "startDate," you should use f-strings rather than adding together separate strings. This will keep things cleaner. 

Please change concatenated strings to f-strings for lines 217-271, 278-302, and 56-68 of this file (vantage_synthetic).